plugins/modules/pfsense_default_gateway.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Orion Poplawski <orion@nwra.com>
+# Copyright: (c) 2018, Frederic Bor <frederic.bor@wanadoo.fr>
+# Copyright: (c) 2023, Nicolas Zagulajew <github@xoop.org>
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: pfsense_default_gateway
+version_added: 0.6.0
+author: "Nicolas Zagulajew (@freeeflyer)"
+short_description: Manage pfSense default gateway
+description: Check and update pfSense default gateway
+notes:
+options:
+  name:
+    description: Default gateway name
+    required: false
+    type: str
+  ipprotocol:
+    description: Choose the Internet Protocol Version for this gateway.
+    required: false
+    choices: [ "inet", "inet6" ]
+    default: inet
+    type: str
+"""
+
+EXAMPLES = """
+- name: Sets default gateway to automatic
+  pfsense_default_gateway:
+    name: Automatic
+    ipprotocol: inet
+
+- name: Remove gateway (ie setting it to None)
+  pfsense_default_gateway:
+    name: none
+    ipprotocol: inet
+
+- name: return gateways 
+  pfsense_default_gateway:
+
+"""
+
+RETURN = """
```

Comment:
Need to add `defaultgw6` and `defaultgw4`

Reply:
done